.github/workflows/tests.yaml

```diff
@@ -95,6 +95,8 @@ jobs:
           # Remove newer go and install regular version for 20.04
           sudo snap remove go
           sudo apt install -y golang
+          # Install RPM dependencies for RPM tests
+          sudo apt install rpm
```

Comment:
@cmatsuoka do SourceHandlers have a way to declare dependencies that they need in order to actually handle the sources? For example, how are "7z" files handled?

Reply:
https://github.com/canonical/craft-parts/blob/main/craft_parts/packages/deb.py#L372

Reply:
Do we know if `rpm2cpio` is even a valid source type anymore? It doesn't seem to be, but I hesitate to remove it here. (I've otherwise updated things though)

Reply:
7z sources are currently not handled, but I have an old PR to add it: https://github.com/canonical/craft-parts/pull/232 :cake: